hack/jenkins/common.sh

```diff
@@ -89,6 +89,27 @@ if [ "$(uname)" = "Darwin" ]; then
   fi
 fi
 
+## set sysctl params for inotify, to avoid "too many open files" errors
+## ref: https://cloud.google.com/kubernetes-engine/distributed-cloud/bare-metal/docs/installing/configure-os/ubuntu
+if [ "$OS" == "linux" ]; then
+  SYSCTL=/etc/sysctl.conf
+
+  MAX_USER_INSTANCES=8192
+  MAX_USER_WATCHES=524288
+
+  if grep -q "^[[:blank:]]*fs.inotify.max_user_instances=" "${SYSCTL}"; then
+    sed -i -E "s/fs.inotify.max_user_instances=.*/fs.inotify.max_user_instances=${MAX_USER_INSTANCES}/g" "${SYSCTL}"
```

Comment:
If we set the value it overrides what we have in /etc/sysctl.conf, so we don't really care about it.

This is also not correct since we can have drop in configs in SYSCTL=/etc/sysctl.conf.d. It does not make sense to look in all of them for the settings and replacing the configuration in files created by other components is not a good idea.

Typically you add a new config file with higher number so it would be processed last and override other configs like /etc/sysctl.conf/99-override.conf.